### DISCUSSION TITLE
Bring back Windows Desktop App

### DISCUSSION THREAD

**Discussion by Cupara** [gameserverapp/Platform#546] [Suggestions]
**Describe the issue**
With the way most sites now offer to install their website using progressive web app (PWA). It would only seem like the right move to enable that type of feature on GameServerApp.com as well.

**Section**
Mainly the Dashboard area.

**Screenshots**
![Default-Dashboard-panel-GameServerApp-com](https://github.com/gameserverapp/Platform/assets/7572268/3ab28e50-897e-419a-89f0-42f9ae4a0b10)

**Additional context**
Being able to manage our Dashboard, Servers, Admin Tools, Tasks, Connect, Configure, Manage Add-ons, Community, and Monetizations right from an app on our desktop would be amazing and so fast for people who like to keep these types of websites open through an app. I still think managing your Account should be done through the website itself.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion. I moved it from the Issue section to the suggestions 👍
